scripts/extensions/sams/src/containers/FileUploadModal.tsx

```diff
@@ -87,6 +93,26 @@ export class FileUploadModal<T> extends React.Component<IProps<T>, IState> {
         }
     }
 
+    getInitialItems(): Array<IUploadItem> {
+        const items: Array<IUploadItem> = [];
+
+        if (this.props.initialFiles != null && this.props.initialFiles?.length > 0) {
```

Comment:
Lint was failing without the null check

Reply:
hmm weird, what was the error?

Reply:
It's not an error. `this.props.initialFiles?.length` evaluates to `number | undefined` and it's not valid to compare
`undefined > 0`. It could be solved by using a default -  `if ( (this.props.initialFiles?.length ?? 0) > 0 ) {...}`,
but inside the `if` statement, `forEach` is called on `initialFiles` which still hasn't been checked if it's null or not.

At run-time it's impossible for it to be null, because the length is checked, but the compiler doesn't understand number comparisons and `5 > 4` evaluates to `boolean`, not `true`;

It could be written in a shorter way if no comparison to a number was made:
```
if (this.props.initialFiles?.length) {
    this.props.initialFiles.forEach(...); // compiler understands that this.props.initialFiles is not null
}
```